src/OpenRiaServices.Server.UnitTesting/Framework/Utility.cs

```diff
@@ -4,13 +4,35 @@
 using System.Globalization;
 using System.Linq;
 using System.Linq.Expressions;
+using System.Threading.Tasks;
 
 namespace OpenRiaServices.Server.UnitTesting
 {
     internal static class Utility
     {
         private const int DefaultChangeSetEntryId = 1;
 
+        /// <summary>
+        /// Do a blocking wait on a ValueTask that work even when task is not completed and it is backed by a IValueTaskSource
+        /// Se https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2012
+        /// </summary>
+        public static T SafeGetResult<T>(this ValueTask<T> valueTask)
```

Comment:
Is there any merit to have methods outside of tests?

Reply:
I found 1 usage in server assembly so I considered it easier to just copy to logic.